shells/lib/utils.js

```diff
@@ -3,6 +3,8 @@ import {Arc} from '../../build/runtime/arc.js';
 import {RecipeResolver} from '../../build/runtime/recipe/recipe-resolver.js';
 import {PlatformLoader} from '../../build/platform/loader-web.js';
 import {PecIndustry} from '../../build/platform/pec-industry-web.js';
+import {defaultCoreDebugListeners} from '../../build/runtime/debug/arc-debug-handler.js';
```

Comment:
Would you mind factoring the code that marshals `debugListenerClasses` into a separate file? `lib/debug-listeners.js` would be fine. 

So we end up with something like:
```
import {debugListeners} from './debug-listeners.js'
...
    listenerClasses: debugListeners
```
?

Reply:
I can do this in a separate PR if you rather, not sure what's best.

Reply:
I'll do this and update the PR. You have other fish to fry.